package.json

```diff
@@ -61,8 +67,9 @@
     "typescript": "^4.8.4"
   },
   "dependencies": {
-    "@tableland/sdk": "^4.0.0-pre.4",
-    "@tableland/sqlparser": "^1.0.0",
+    "@ensdomains/ensjs": "3.0.0-alpha.39",
+    "@tableland/sdk": "^4.0.0-pre.6",
+    "@tableland/sqlparser": "1.0.0",
```

Comment:
This will need to be updated, surprised it isn't causing issues already?

Reply:
Wasn't causing issues, but upgraded to 1.0.3